<launch>
    <!-- USB Camera Node -->
    <node name="usb_cam" pkg="usb_cam" type="usb_cam_node" output="screen">
        <param name="video_device" value="/dev/video0" />
        <param name="image_width" value="1280" />
        <param name="image_height" value="720" />
        <param name="pixel_format" value="mjpeg" /> <!-- Adjust if your camera uses a different format -->
        <param name="camera_frame_id" value="usb_cam" />
        <param name="io_method" value="mmap"/>
        <param name="framerate" value="30" />
    </node>

    <!-- Color Target Detector Node -->
    <node name="color_calibrator" pkg="color_target_tracker" type="color_calibrator.py" output="screen" />
</launch>
